src/main/scala/mesosphere/marathon/core/task/termination/impl/KillServiceActor.scala

```diff
@@ -97,10 +97,11 @@ private[impl] class KillServiceActor(
     processKills()
   }
 
+  //todo test this
```

Comment:
Why not add the test right now?

Reply:
either test it or drop the log line (with strong preference to the first choice). I really don't think these TODOs are helpful in the code in long run...